.github/workflows/test-and-publish-nightly.yml

```diff
@@ -28,64 +25,53 @@ jobs:
   build-binaries-and-test:
     strategy:
       matrix:
-        runson:
-          - display: linux-x64
-            name: ubuntu-latest # Using "latest" here as the build and test will any ways run inside a container which we control
```

Comment:
Having `ubuntu-latest` in the gha / artifact name instead of `linux-x64` is confusing. It doesn't matter where it was built, especially when it is built in docker on `ubuntu-latest`:

```
Test and Publish Nightly Builds / test (ubuntu-24.04-arm, corretto 11 arm64 (pull_request)
Test and Publish Nightly Builds / test (ubuntu-latest, corretto 11 x64 (pull_request)
```

Reply:
Good point!

Reply:
I guess we need to keep the display name then